oaknational/Oak-Web-Application
# Feat/lesq 242/lesson header update [LESQ-242]

## Description

- Added a new component `HeaderMetadata` to display the extra data in lesson listing header
- Updated graphql queries with the updated schema

## How to test

1. Go to https://deploy-preview-1955--oak-web-application.netlify.thenational.academy/teachers/programmes/maths-secondary-ks4-foundation/units/surds/lessons
2. You should see the year and tier in the header

## Screenshots

How it should now look:
<img width="500" alt="Screenshot 2023-09-29 at 13 23 38" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/8b5c2a61-9e73-42c5-82b8-e5117e6f504c">
<img width="500" alt="Screenshot 2023-09-29 at 13 22 14" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/51fc638f-bc05-497a-a6ea-a20ddba5f851">
<img width="500" alt="Screenshot 2023-09-29 at 14 00 22" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/c3076f80-df04-4498-8f08-5439c46cab9e">


## Checklist

- [ ]  A lesson listing page should show the year group a unit belongs to in the header
- [ ]  A lesson listing page should show the tier and exam board a unit belongs to IF it has a tier and exam board
